packages/nextjs/components/projects/Card.tsx

```diff
@@ -6,7 +6,7 @@ const Card = ({ project, display }: any) => {
   return (
     <div className="">
       {display === "grids" ? (
-        <div className="border rounded-[1.5rem] border-gray-300  p-4 ">
```

Comment:
The border is in the Figma so I think we should add it back in. https://www.figma.com/file/LLwuijWZJC6bhmgE284VFB/%5BVit%5D---RetroPGF---Voting-(Copy)?type=design&node-id=692-70735&mode=design&t=906GAsJylpFKWhts-4